src/legacy/ui/public/field_editor/field_editor.js

```diff
@@ -80,6 +80,9 @@ import { copyField, getDefaultFormat, executeScript, isScriptValid } from './lib
 
 import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
 
+// This loads Ace editor's "groovy" mode, used below to highlight the script.
+require('brace/mode/groovy');
```

Comment:
nit: Would `import 'brace/mode/groovy';` work here instead of `require`?

Reply:
++ `import 'brace.mode.groovy';`

Reply:
Done, using `import`.